test_launch_ros/test/test_launch_ros/actions/test_ros_timer.py

```diff
@@ -114,9 +114,9 @@ def test_timer_action_sanity_check():
     assert 0 == ls.run()
     assert shutdown_reasons[0].reason == 'One second timeout'
 
-    # Verify that 1 sec has passed between start of test and timeout
-    tolerance = 0.1
-    assert (end_time - start_time) > 1 - tolerance
+    # Verify that at least 1 sec has passed between start of test and timeout
+    tolerance = 0.25
```

Comment:
Honestly, I'd just make this 1.0.  Even a full second here is not *always* going to succeed (particularly on Windows), but it should be a lot less flaky that way.

Otherwise this refactor makes sense to me.

Reply:
Yeah, I've never seen this error before, so I imagine just increasing the tolerance was a good fix.

I don't have a strong opinion, so I can remove the test if you think that'd be better long term speaking.

Reply:
> Yeah, I've never seen this error before, so I imagine just increasing the tolerance was a good fix.

I'd say let's start with a tolerance of 1.0, and the rest of what you've done here.  If the test continues to be a problem, we can remove it later on.

Reply:
Sure, added with: 1f02915